### ISSUE TITLE
PreferToOverPairSyntax false positive

### ISSUE THREAD

**Issue by vanniktech** [arturbosch/detekt#1066]
Android has it's own Pair class. https://developer.android.com/reference/android/util/Pair

That one you can't construct using `1 to 2`. Hence the following code should not get flagged:

```kotlin
android.support.v4.util.Pair(Location(0.0, 0.0), 200.0)
```

**Comment by arturbosch** [comment]
That's tricky without symbol solving. But we could check the import for that class ^^

**Comment by vanniktech** [comment]
I wasn't. Probably just a leftover from converting Java to Kotlin class. I don't think the generator changes the Pair class.

**Comment by vanniktech** [comment]
But yeah I agree that the Kotlin one should be used. Maybe there's still an API that requires the Android one to be called. In any case it should not be flagged.
